python/pyspark/sql/window.py

```diff
@@ -212,16 +218,20 @@ def rangeBetween(self, start, end):
         values directly.
 
         :param start: boundary start, inclusive.
-                      The frame is unbounded if this is ``Window.unboundedPreceding``, or
+                      The frame is unbounded if this is ``Window.unboundedPreceding``,
+                      ``org.apache.spark.sql.catalyst.expressions.UnboundedPreceding``, or
                       any value less than or equal to max(-sys.maxsize, -9223372036854775808).
         :param end: boundary end, inclusive.
-                    The frame is unbounded if this is ``Window.unboundedFollowing``, or
+                    The frame is unbounded if this is ``Window.unboundedFollowing``,
+                    ``org.apache.spark.sql.catalyst.expressions.UnboundedFollowing``, or
```

Comment:
Wait .. why do we expose `org.apache.spark.sql.catalyst` path in Python doc .. ? In addition, this package is meant to be internal if I haven't missed something .. ?